libraries/AC_AttitudeControl/AC_AttitudeControl.cpp

```diff
@@ -150,6 +150,27 @@ const AP_Param::GroupInfo AC_AttitudeControl::var_info[] = {
     // @User: Standard
     AP_GROUPINFO("INPUT_TC", 20, AC_AttitudeControl, _input_tc, AC_ATTITUDE_CONTROL_INPUT_TC_DEFAULT),
 
+    // @Param: LAND_R_MULT
+    // @DisplayName: Roll gain multiplier used when landed
+    // @Description: Roll gain multiplier used when landed
```

Comment:
We should give some instructions on what lower vs higher values do.  E.g. "lower values result in lower gains during landing, higher values result in higher gains during landing"

Reply:
Does this work?
    // @Description: Roll gain multiplier used when landed. A factor of 1.0 means no reduction in gain while landed. Reduce this factor to reduce ground oscitation in the roll axis.